### DISCUSSION TITLE
GPIO as an input script

### DISCUSSION THREAD

**Discussion by honey-pablo** [josephdadams/TallyArbiter#740] [General]
Hi,

I'm trying to use the GPIO input script that presents to Tally Arbiter as an OSC source.

I've configured my GPIO pins and server IP Port.

Whenever I run the script as soon as I take a GPIO pin to ground I get the following error

SimpleUDPClient.send_message() missing 1 required positional argument: 'value'

Any thoughts?  thanks!

**Comment by josephdadams** [comment]
Which script is this?

**Reply by honey-pablo** [reply]
https://gist.github.com/josephdadams/889bac1bcbc1d48d0f0d496ba6c4d44f

**Comment by honey-pablo** [comment]
To answer my own question - line 36 in the example script has a missing comma.

**Comment by josephdadams** [comment]
Fixed it there!
